release-notes/v5.5.0.md

```diff
@@ -111,3 +111,7 @@
 #### <sub><sup><a name="v550-note-28" href="#v550-note-28">:link:</a></sup></sub> fix
 
 * @xavierzwirtz also fixed `hg.pull` by passing the `sourceUri` in the Mercurial resource. This could fix some issues where Mercurial wasn't saving the full uri in the `hgrc` file concourse/hg-resource#13.
+
+#### <sub><sup><a name="v550-note-29" href="#v550-note-29">:link:</a></sup></sub> fix
+
+* Scrollbars have been given a touch of the Concourse design language. It should no longer use browser defaults.
```

Comment:
Cute! Can you mention this PR number (#4355) in the note? I wonder if this will be a recurring problem

Reply:
Might be worth calling out `Scrollbars in Chrome` (and maybe FF if you add the above note's consideration to the PR) so that if there are Safari or Opera or whatever users out there they don't get confused.

Reply:
Maybe we can build a cli tool for this we do to autogen migrations

Reply:
🤔 i like the sounds of that

Reply:
but @matthewpereira's point is a good one. I'm not sure that Opera or Safari are going to support the same bleeding-edge features that Firefox does.